sdk/openai/openai-assistants/sources/customizations/rest/models.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We don't normally customize the REST definitions, could you share more what is being customized here?